docs/cli/README.md

```diff
@@ -0,0 +1,482 @@
+$ altis-cli config
+
+Configuration commands
+
+Commands:
+  altis-cli config reset   Reset configuration
+  altis-cli config setup   Set up configuration
+  altis-cli config status  Show stored configuration
+```
+
+### Config setup
+
+```sh
+$ altis-cls config setup
+Welcome to altis-cli!
+? Run initial setup? Yes
+Saved configuration!
+Run altis-cli config setup at any time to run setup.
+```
+
+This will run the Altis Dashboard in your default browser and prompt to log in to the dashboard. It creates an authentication
+key and stores it in the `config.json` file.
+
+![Screenshot of Altis Desktop Authentication](../assets/altis-cli-authenticate.png)
+
+Once you are authenticated, you see this screen
+
+![Screenshot of Altis Desktop Authenticated](../assets/altis-cli-authenticated.png)
+
+### Config reset
+
+Running config reset will clear the authentication token from the configuration file.
+
+### Config status
+
+Running config status will list the username you are connected to Altis Dashboard with.
+
+## Stack commands
+
+Altis CLI allows you to interact with the stacks you have access to. The following commands are available:
+
+```sh
+
+Commands:
+  altis-cli stack backup [stack]       Create a new backup for the stack.
+  altis-cli stack backups [stack]      List backups for the stack.
+  altis-cli stack deploy [stack]       Deploy a given stack.
+  altis-cli stack info [stack]         Get information for a stack.
+  altis-cli stack list                 List stacks available in our hosting.
+  altis-cli stack local-setup [stack]  Setup an existing Altis stack locally.
+  altis-cli stack php-logs [stack]     Show PHP logs for a stack.
+  altis-cli stack scp <src> <dest>     Copy a file to/from a stack. Remote
+                                       src/dest are determined by a colon in the
+                                       input, e.g. stack:/usr/src/app.
+  altis-cli stack ssh [stack]          SSH into a stack.
+```
+
+### Stack backup
+
+The `stack backup` command is used to create and download backups for your application. Note the default is to include both database
+and downloads in the backup.
+
+```sh
+altis-cli stack backup [stack]
+
+Create a new backup for the stack.
+
+Options:
+  --version   Show version number                                      [boolean]
+  --help      Show help                                                [boolean]
+  --uploads   Include uploads in the backup.
+              (--no-uploads to disable)                [boolean] [default: true]
+  --database  Include database in the backup.
+              (--no-database to disable)               [boolean] [default: true]
+  --resume    Log ID for resuming an existing deploy.                   [string]
+  --debug     Enable internal debugging information.  [boolean] [default: false]
+```
+
+For example, to create a database only backup, you would run the following command:
+
+```sh
+bin/altis-cli stack backup --database --no-downloads platform-test
```

Comment:
We have `bin` in some example. I'd remove that as we say above that you should run it via `npx` or the globally installed command